spec/factories/users.rb

```diff
@@ -0,0 +1,16 @@
+FactoryGirl.define do
+  factory :user do
+    email 'test@test.by'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.